src/base/QXmppRosterIq.h

```diff
@@ -108,6 +122,8 @@ class QXMPP_EXPORT QXmppRosterIq : public QXmppIq
     QList<Item> m_items;
     // XEP-0237 Roster Versioning
     QString m_version;
+    // XEP-0405: Mediated Information eXchange (MIX): Participant Server Requirements
```

Comment:
Same as above: let's move everything behind a d-pointer.

Reply:
I can do that after https://github.com/qxmpp-project/qxmpp/pull/189 has been merged.

Reply:
done